crates/net/ecies/src/algorithm.rs

```diff
@@ -730,6 +730,12 @@ impl ECIES {
         self.ingress_aes.as_mut().unwrap().apply_keystream(ret);
         Ok(split_at_mut(ret, size)?.0)
     }
+
+    /// return num alignment value
+    #[inline]
+    const fn align_num(num: usize, align: usize) -> usize {
+        (num + (align - 1)) & !(align - 1)
+    }
```

Comment:
@DaniPopes need help here 🙃, is this euqivalent

Reply:
this should be equal to "<integer>.next_multiple_of"